backend/test/app.js

```diff
@@ -4,6 +4,7 @@ const assert = require('assert');
 const App = require('../src/app');
 const Usuario = require('../src/modelos/usuario');
 const RepositorioUsuarios = require('../src/repositorios/repositorioUsuarios');
+const repositorioDeSolicitudes = require('../src/repositorios/repositorioDeSolicitudes');
```

Comment:
```suggestion
const RepositorioSolicitudes = require('../src/repositorios/repositorioDeSolicitudes');
```

Lo mismo que más arriba, para mantener la misma convención por así decirlo